.travis.yml

```diff
@@ -2,7 +2,7 @@ language: dart
 
 dart:
 - dev
-- 2.0.0
+- stable
```

Comment:
Please don't! Try to keep this at the latest supported SDK. So 2.1.1

Reply:
Done.